packages/orchid/src/__tests__/example-task.ts

```diff
@@ -1,79 +1,53 @@
-import { TaskSpec, TaskContext, SpecToTask } from '../task.types';
-
-declare global {
-  interface GlobalTasks {
-    main: SpecToTask<typeof mainTask>;
-    double: SpecToTask<typeof doubleTask>;
-    list: SpecToTask<typeof listTask>;
-    addOne: SpecToTask<typeof addOneTask>;
-    sum: SpecToTask<typeof sumTask>;
-  }
-}
+import { TaskSpec } from '../task.types';
 
 export const mainTask: TaskSpec<undefined, Promise<number>> = {
-  name: 'main',
-  async run(input: undefined, { run, log }: TaskContext) {
-    const list = await run('list', 4);
+  id: 'main',
+  async run(input: undefined, { run, log }) {
```

Comment:
How about switching the arguments order? You probably always need the context but in this case you don't need the input.

Reply:
I thought about it, but there are other cases, when its a leaf and i only need the input but not the context. 
I dont have a strong preference.
It is just kinda hard to predict which one is less likely.